source/sessions/snapshot-sessions.rst

```diff
@@ -0,0 +1,339 @@
+============================
+Snapshot Reads Specification
+============================
+
+:Spec Title: Snapshot Reads Specification (See the registry of specs)
+:Spec Version: 1.0
+:Author: Boris Dogadov
+:Advisors: Jeff Yemin, A. Jesse Jiryu Davis, Judah Schvimer
+:Status: Draft (Could be Draft, Accepted, Rejected, Final, or Replaced)
+:Type: Standards
+:Minimum Server Version: 5.0
+:Last Modified: 15-Jun-2021
+
+.. contents::
+
+--------
+
+Abstract
+========
+
+Version 5.0 of the server introduces support for readConcern level "snapshot" (non-speculative)
+for read commands outside of transactions, including on secondaries.
+This spec builds upon the Sessions Specification to define how an application
```

Comment:
Could do a relative link to the [driver-sessions](https://github.com/mongodb/specifications/blob/master/source/sessions/driver-sessions.rst) specification here.